gitreceive/receiver/flynn-receive.go

```diff
@@ -189,23 +199,25 @@ Options:
 			Processes: map[string]int{"web": 1},
 		}
 
-		watcher, err := client.WatchJobEvents(app.ID, release.ID)
-		if err != nil {
-			log.Fatalln("Error streaming job events", err)
-			return
-		}
-		defer watcher.Close()
```

Comment:
Thinking of adding parameter `logf func(*ct.DeploymentEvent)` to `DeployAppRelease`. This will be `nil` on other call sites and specialized in this case. Also remove the timeout all together?